articles/how-to-setup-ci-cd-to-vue-with-firebase-hosting-and-github-actions/index.md

```diff
@@ -2,264 +2,212 @@
 layout: engineering-education
 status: publish
 published: true
-url: /engineering-education/how-to-setup-ci-cd-to-vue-with-firebase-hosting-and-github-actions/
-title: How to setup CI/CD to Vue with Firebase Hosting and Github Actions
-description: Learn How to setup CI/CD to Vue with Firebase Hosting and Github Actions.
+url: /engineering-education/ci-cd-setup-vue/
+title: How to  Set Up CI/CD in a Vue Project with Firebase Hosting and Github Actions
+description: In this tutorial, we walk through how to set up CI/CD in a Vue Project with Firebase Hosting and Github Actions.
 author: worawat-kaewsanmaung
 date: 2020-04-24T00:00:00-07:00
-topics: []
+topics: [Vue,CI/CD]
 excerpt_separator: <!--more-->
 images:
 
-  - url: /engineering-education/how-to-setup-ci-cd-to-vue-with-firebase-hosting-and-github-actions/hero.jpg
-    alt: How to setup CI/CD to Vue with Firebase Hosting and Github Actions
+  - url: /engineering-education/ci-cd-setup-vue/hero.png
+    alt: CI CD VUE 
 ---
-In the present software development context, the important workflows for software development are run, tested, built, and deployed without setting up CI/CD. All the processes in the software development are done manually from running tests to running build to get the build file and deploying on the server. The development cycle for adding new features and fixing bugs done manually becomes inefficient and hectic due to processes being run repeatedly.
-<!--more-->
+The technological advancement revolves around the automation of every work that is done manually till today. Such automation can be applied in the software development cycle and project management as well. Most of the workflow includes running, testing, building, and deploying the project manually. Each time we have any changes or updates we push the code to our version control system and deploy it manually. In any development phase, it can be measured inefficient to fix bugs and update new features manually as it consumes a lot of time and accuracy. Well, what if we can automate this process of deployment? Yes, we can do that by applying the CI/CD (Continuous Integration / Continuous Delivery) to our development projects. And here, we are going to do that using Vue, Firebase Hosting, and GitHub workflows. The overall process automates the process of software development and management and makes it easy to test and deploy the final results. The bottom line is to automatically build and deploy the updates to the main server after each pull request or change has been merged to the master branch of our GitHub repository. Hence, the overall process involves:
```

Comment:
Please add a title to the article.
The first sentence is extremely lacking, mostly because you seem to have deleted the first part, Either remove the first part of the paragraph, or rephrase it so it sounds like an introductory statement.

Also, the dialog part of the article, while it is not wrong, It is unnecessary.

Replace "The bottom line" with something else like "the end goal"

Reply:
@katungi Hi, thank you for reviewing my post. I didn't understand on "Please add a title to the article"  
which part that you're referring to? in the front matter section?

Reply:
Ignore that, I realized you have the front matter setup. 
Have you run your article through grammarly or hemmingway?

Reply:
@katungi yep also made an update as your comment

Reply:
I would prefer if you returned the original starting paragraph. It explains the context of the article very well.

this paragraph ;

> In the present software development context, the important workflows for software development are run, tested, built, and deployed without setting up CI/CD. All the processes in the software development are done manually from running tests to running build to get the build file and deploying on the server. The development cycle for adding new features and fixing bugs done manually becomes inefficient and hectic due to processes being run repeatedly.

Reply:
@katungi done, I just brought back the old intro